### PR TITLE
NAS-102047: Add help text to ACL Manager fields

### PR DESCRIPTION
**Pull request by Mrt134**
- Fix a couple help text links in dataset-acl.component.ts
- Yarn build testing: links to 11.3 documentation are broke, but that is expected until that guide is published.

### CODE REVIEW

File: src/app/helptext/storage/volumes/datasets/dataset-acl.ts [hunk 1]
```diff
@@ -101,25 +122,28 @@ dataset_acl_advanced_flags_options: [
...
 dataset_acl_recursive_placeholder: T('Apply permissions recursively'),
-dataset_acl_recursive_tooltip: T('Apply permissions recursively to all directories\
- and files within the current dataset'),
+dataset_acl_recursive_tooltip: T('Apply permissions recursively to all\
+ directories and files in the current dataset.'),
 
 dataset_acl_traverse_placeholder: T('Apply permissions to child datasets'),
-dataset_acl_traverse_tooltip: T('Apply permissions recursively all child datasets\
- within the current dataset'),
+dataset_acl_traverse_tooltip: T('Apply permissions recursively all child\
```
Comment (wblock) [inline src/app/helptext/storage/volumes/datasets/dataset-acl.ts]:
```suggestion
dataset_acl_traverse_tooltip: T('Apply permissions recursively to all child\
```

File: src/app/helptext/storage/volumes/datasets/dataset-acl.ts [hunk 2]
```diff
@@ -7,50 +7,63 @@ dataset_acl_path_placeholder: T('Path'),
...
+ appear in the drop-down menu.'),
 
 dataset_acl_gid_placeholder: T('Group'),
-dataset_acl_gid_tooltip: T('Select the group to control the pool/dataset. Groups\
- manually created or imported from a directory service\
- will appear in the drop-down menu.'),
+dataset_acl_gid_tooltip: T('The group which controls the dataset. This\
+ group always has permissions to read or write the ACL and read or write\
+ attributes. Groups created manually or imported from a directory\
+ service appear in the drop-down menu.'),
```
Comment (anodos325) [inline src/app/helptext/storage/volumes/datasets/dataset-acl.ts]: group@ does not always have permissions to read or write the ACL.
```
root@S4:/mnt/SHARE` # getfacl STUFF
# file: STUFF
# owner: root
# group: smbuser
            owner@:rwxp--aARWcCos:-------:allow
            group@:r-x---a-R-c--s:-------:allow
         everyone@:r-x---a-R-c--s:-------:allow
root@S4:/mnt/SHARE # su smbuser
$ setfacl -x 2 STUFF
setfacl: STUFF: acl_set_file() failed: Operation not permitted
```

Comment (Mrt134) [reply src/app/helptext/storage/volumes/datasets/dataset-acl.ts]: Aha, thanks for the clarification!

**Comment by wblock** [comment]
NAS-101732 (also affects).

**Pull request merged**